recipes/astrometry.net/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "astrometry.net" %}
+{% set version = "0.78" %}
+
+package:
+  name: {{ name|lower}}
+  version: {{ version }}
+
+source:
+  fn: {{name}}-{{ version }}.zip
+  url: https://github.com/dstndstn/astrometry.net/archive/{{ version }}.zip
+  sha256: 85480b9636757c89ef5ccc7aac557409ac2ab377eb6c6eca862b16cdd4faefd9
+
+build:
+  number: 0
+  skip: True  # [osx or win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - swig
+    - git
+    - make
+    - numpy
+    - zlib
+    - jpeg
+    - netpbm
+    - libpng
+    - cfitsio
+    - astropy
+    - bzip2
+    - cairo
+    - gsl
```

Comment:
most of these are not build deps

Reply:
and are instead host deps

Reply:
Changed them to host

Reply:
However, with them in host, we have an additional problem: astrometry.net has a header file called `tic.h`, which conflicts with ncurses `tic.h`. With these libraries in build, we can remove the ncurses header easily. But, with these dependencies on host, how can we remove this header file?

Reply:
Seems to be ok now

Reply:
@scopatz can you review this package again?